blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DBasicNumericIO.java

```diff
@@ -0,0 +1,33 @@
+package com.rackspacecloud.blueflood.io.datastax;
```

Comment:
I guess...I was trying to follow your convention with no Rollups in the name. :-P